compiler/passes/ResolveScope.cpp

```diff
@@ -1095,6 +1096,8 @@ ResolveScope::lookupPublicUnqualAccessSyms(const char* name,
                                            BaseAST *context,
                                            std::map<Symbol*,
                                              astlocT*>& renameLocs,
+                                           std::map<Symbol*,
+                                             VisibilityStmt*>& reexportPts,
```

Comment:
I would ditch aligning the first column of formals and just start two indents inside in the next line.